Mathlib/MeasureTheory/Constructions/Pi.lean

```diff
@@ -903,6 +931,28 @@ theorem volume_preserving_pi {α' β' : ι → Type*} [∀ i, MeasureSpace (α'
     MeasurePreserving (fun (a : (i : ι) → α' i) (i : ι) ↦ (f i) (a i)) :=
   measurePreserving_pi _ _ hf
 
+/-- Docstring. -/
```

Comment:
Yes please, unless you're only adding it because `arrowCongr'` ends in a `'`, in which case you can use the `set_option` locally

Reply:
Oh, somehow, the docstrings got lost on the way 😅 I'll fix that. Thanks!

Reply:
I put some more descriptive docstrings.